editor/osm_editor.cpp

```diff
@@ -646,8 +646,10 @@ void Editor::UploadChanges(string const & oauthToken, ChangesetTags tags,
               }
               else
               {
-                LOG(LDEBUG, ("Create case: uploading patched feature", osmFeature));
-                changeset.Modify(osmFeature);
+                LOG(LDEBUG, ("Create case: uploading new feature", feature));
+                // There is another node nearby, but it is saver to upload a new node (#2298).
+                changeset.Create(feature);
+                changeset.AddChangesetTag("info:feature_close_by", "yes");
```

Comment:
This line should go _before_ the previous line to reliably send tags to the server when changeset is created, not when it's closed.